libs/pages/cluster/src/lib/ui/container/container.tsx

```diff
@@ -18,48 +18,43 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
   const statusLoading = !!cluster?.extendedStatus?.status?.status
 
   const headerActions = (
-    <>
-      <Skeleton width={150} height={32} show={!statusLoading}>
+    <div className="flex flex-col gap-3">
+      <div className="flex flex-row gap-2">
+        {cluster?.production && (
+          <Badge size="xs" color="neutral">
+            PROD
+          </Badge>
+        )}
+        {cluster?.is_default && (
+          <Badge size="xs" color="sky">
+            DEFAULT
```

Comment:
Same here `Default`, should be more ISO with other Badge
<img width="523" alt="Capture d’écran 2023-09-26 à 15 15 59" src="https://github.com/Qovery/console/assets/533928/fea42ffe-f33c-4014-9f91-b40b45a725e8">

Reply:
https://github.com/Qovery/console/pull/850/commits/d1690f90b3bd575d616cfe5671ce54442e5749e9